kiwi/xml_parse.py

```diff
@@ -16,7 +16,7 @@
 #   kiwi/schema/kiwi_for_generateDS.xsd
 #
 # Command line:
-#   /home/ms/Project/kiwi/.tox/3.6/bin/generateDS.py -f --external-encoding="utf-8" --no-dates --no-warnings -o "kiwi/xml_parse.py" kiwi/schema/kiwi_for_generateDS.xsd
+#   /home/david/work/kiwi/.env3/bin/generateDS.py -f --external-encoding="utf-8" --no-dates --no-warnings -o "kiwi/xml_parse.py" kiwi/schema/kiwi_for_generateDS.xsd
```

Comment:
One thing we should check is that we don't re-introduce the issue solved in (2ea7fbd13f13487)

@davidcassany can you make sure generateDS version is v2.29.14 ? Thanks

Reply:
@scheafi yes it is, see 5th line :smile: Yes I remember this ugly issue...

Reply:
:+1: